bin/blocks-plot

```diff
@@ -34,14 +34,14 @@ def plot_dataframe(dataframe):
     print("Plotting {} channels:".format(len(dataframe.columns)))
     for cname, series in iteritems(dataframe):
         print("    {}".format(cname))
-        pylab.plot(t, series.interpolate(), label=cname)
```

Comment:
Ok, using interpolate('nearest') fixes this. 

I also added a unit test that ensures the interpolation method has the desired properties.